tests/main/upgrade-sudoers-conffile-removal/task.yaml

```diff
@@ -0,0 +1,27 @@
+summary: Ensure conffile removal from upgrades <2.46 works
```

Comment:
We have the tests/upgrade to heck upgrades, does it make sense to move this test there?